### ISSUE TITLE
Add new sensor field

### ISSUE THREAD

**Issue by ituri** [opendata-stuttgart/sensor.community#209]
Hi,

I registered a new device (multigeiger) which originally only had the geiger tube installed. Thus, I removed the second sensor field. Now I added a BME680 to the device but it seems like I can't add the sensor in the device settings on the website. Also, I don't want to delete and re-add the sensor since I can't add another sensor with the same sensor id after that.
What can I do here?

**Comment by ricki-z** [comment]
Hello,
are you sure that you've added a BME680? Out database doesn't support this sensor, only BMP180, BMP280 and BME280.

**Comment by ituri** [comment]
Yes, I am. I was inspired by the instructions on [heise.de](https://www.heise.de/hintergrund/Details-und-Nachbautipps-zum-MultiGeiger-von-Ecocurious-7099642.html) where the author also used a BME680. My sensor is definitely a BME680 and it's also reporting the correct values - I checked with my weather station and other thermometers/barometers around.

But the sensor itself is not the issue. It's just the fact that I deleted one of the two sensors initially and now I can't add it back. How can I get the values of the Geiger tube back again?

**Comment by ricki-z** [comment]
could you please send me the chipID to tech (@) sensor.community? Then I will add a dummy entry for an BME280. The firmware should be changeable to only send the BME680 values for pressure, temperature and humidity to our API server.

**Comment by ituri** [comment]
Done!
